website/static/js/commentpane.js

```diff
@@ -11,7 +11,10 @@
         onOpen: function() {}
     };
 
-    var CommentPane = CommentPane || function(selector, options) {
+    /*
+       mode: 'widget', 'pane' or 'page'
+     */
+    var CommentPane = CommentPane || function(selector, hostPage, hostName, mode, options) {
```

Comment:
I think `hostPage`, `hostName`, and  `mode` are outside the responsibility of `CommentPane`. Can they be removed here?